services/site/src/styles/custom.scss

```diff
@@ -68,6 +68,17 @@
     @apply inline-block;
 }
 
+.instruction-text a {
```

Comment:
The styling for markup content is managed via tailwind typography plugin. They styles are applied via the `prose` class. The config for that is in the [tailwind config typography section](https://github.com/a11yphant/a11yphant/blob/42531264e3f3e42c3b35bcc1c4ce4edc10442c52/services/site/tailwind.config.js#L68). We can adapt the styling for the a tags there. Then this will automatically also apply for the other places where we use markup for content.

Reply:
That's a good point! I moved the styles there